src/libs/antares/study/parts/hydro/container.cpp

```diff
@@ -330,6 +332,12 @@ bool PartHydro::LoadFromFolder(Study& study, const AnyString& folder)
               && ret;
     }
 
+    if (IniFile::Section* section = ini.find("overflow cost"))
+    {
+        ret = loadProperties(study, section->firstProperty, buffer, &PartHydro::overflowCost)
+              && ret;
+    }
+
     return ret;
```

Comment:
I assume that levelCost also need to be loaded. If not, levelCost will be always 0, since it is assigned to 0 in class definition.

Reply:
4189a3e96